pom.xml

```diff
@@ -53,9 +53,14 @@
         <version.org.sonatype.plugins.nexus-staging-maven-plugin>1.6.6</version.org.sonatype.plugins.nexus-staging-maven-plugin>
         <version.checkstyle>8.5</version.checkstyle>
         <version.org.wildfly.checkstyle-config>1.0.7.Final</version.org.wildfly.checkstyle-config>
+        <version.org.jacoco>0.7.8</version.org.jacoco>
```

Comment:
@zhfeng Is it a hard dependency for some Arquillian reason? I've been using 0.8.2...